app/controllers/concerns/versioning.rb

```diff
@@ -0,0 +1,36 @@
+module Versioning
```

Comment:
Good question, I've had a long think about this, the regex we are using to extract the version isn't vendor API specific, and everything seems generic, I think I prefer to keep it outside that namespace as it's easier to add elsewhere when/if the time comes. If we were to ever add anything specific to the vendor API happy to move into that namespace otherwise.